pom.xml

```diff
@@ -103,11 +103,11 @@
     <repository>
       <uniqueVersion>false</uniqueVersion>
       <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases</url>
+      <url>http://repo.jenkins-ci.org/releases/</url>
```

Comment:
Is there a compelling reason not to use HTTPs by default here? https://repo.jenkins-ci.org has a proper certificate and everything